squad/frontend/project_settings.py

```diff
@@ -35,6 +41,24 @@ def settings(request, group, project):
     return render(request, 'squad/project_settings/index.jinja2', context)
 
 
+@auth_write
+def advanced_settings(request, group, project):
+    if request.method == "POST":
+        form = ProjectFormAdvanced(request.POST, instance=request.project)
+        if form.is_valid():
```

Comment:
will this use yaml_validator?

Reply:
Yes, it throws a warning like `Dictionary object expected` when it doesn't match yaml syntax